com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.hlsl

```diff
@@ -391,74 +385,99 @@ void LightLoop( float3 V, PositionInputs posInput, PreLightData preLightData, BS
         }
 #endif
 
+        // Explanation about APV and SSGI/RTGI/Mixed effects steps in the rendering pipeline.
+        // All effects will output only Emissive inside the lighting buffer (gbuffer3) in case of deferred (For APV this is done only if we are not a lightmap).
+        // The Lightmaps/Lightprobes contribution is 0 for those cases. Code enforce it in SampleBakedGI(). The remaining code of Material pass (and the debug code)
+        // is exactly the same with or without effects on, including the EncodeToGbuffer.
+        // builtinData.isLightmap is used by APV to know if we have lightmap or not and is harcoded based on preprocessor in InitBuiltinData()
+        // AO is also store with a hack in Gbuffer3 if possible. Otherwise it is set to 1.
+        // In case of regular deferred path (when effects aren't enable) AO is already apply on lightmap and emissive is added on to of it. All is inside bakeDiffuseLighting and emissiveColor is 0. AO must be 1
+        // When effects are enabled and for APV we don't have lightmaps, bakeDiffuseLighting is 0 and emissiveColor contain emissive (Either in deferred or forward) and AO should be the real AO value.
+        // Then in the lightloop in below code we will evalaute APV or read the indirectDiffuseTexture to fill bakeDiffuseLighting.
+        // We will then just do all the regular step we do with bakeDiffuseLighting in PostInitBuiltinData()
+        // No code change is required to handle AO, it is the same for all path.
+        // Note: Decals Emissive and Transparent Emissve aren't taken into account by RTGI/Mixed.
+        // Forward opaque emissive work in all cases. The current code flow with Emissive store in GBuffer3 is only to manage the case of Opaque Lit Material with Emissive in case of deferred
+        // Only APV can handle backFace lighting, all other effects are front face only.
+
+        // If we use SSGI/RTGI/Mixed effect, we are fully replacing the value of builtinData.bakeDiffuseLighting which is 0 at this step.
+        // If we are APV we only replace the non lightmaps part.
+        bool replaceBakeDiffuseLighting = false;
+#if defined(PROBE_VOLUMES_L1) || defined(PROBE_VOLUMES_L2)
+        float3 lightInReflDir = float3(-1, -1, -1); // This variable is used with APV for tinting reflection probe - see code for LIGHTFEATUREFLAGS_ENV
```

Comment:
Tiny note but I wouldn't refer to the normalization as tinting as it might be confusing to the reader of the code.

Reply:
udpated